backend/siarnaq/api/episodes/models.py

```diff
@@ -280,155 +278,53 @@ def __str__(self):
     def get_potential_participants(self):
```

Comment:
`potential` seems to imply that not everyone in here will actually end up as a participant. Want to rename? eg. to `get_eligible_teams`?